Misc Updates and Initial Gloval Nav Bar Code

stevemanone opened the pull request:
This is a VERY messy hodgepodge of misc changes (most are last minute updates requested by Anandana for various presentations).  The initial implementation of the nav bar code is solely in the App.js file (the code is very basic and does not have logic for a context sensitive help menu/remnant icon appearing after video views).

stevemanone committed:
Updated acknowledgements

stevemanone committed:
Merge remote-tracking branch 'upstream/master'

stevemanone committed:
Misc changes

Changes are mostly for 4/24 demonstration and including a very messy way
to 'disable' certain videos (videos do not lead to youtube player when clicked,
and only specific videos are allowed).

stevemanone committed:
Misc changes done to allow compatibility with 5/3 demo phone.

This includes creating a smaller (less pixels) version of the App
Landing page (original picture would not show up on the small phone
screen) and slighly changing the logic used to hide the remenant
(setting opacity to 0 seemed to only dim but not completely hide the
remnant icon).

stevemanone committed:
Updated .gitignore

JacksBurden committed:
First pass at object chooser pages

bdougsand committed:
First stab at creating the Help page

bdougsand committed:
Scale fonts by pixel ratio

This will need revisiting, but getFontScale() did not work as expected.

bdougsand committed:
Added icons and play button to Help section headers

bdougsand committed:
Help page: Show a darker icon while a sound is playing

bdougsand committed:
Begin button animation on press in instead of after release

stevemanone committed:
Initial Commit of Nav Bar

Very preliminary version of nav bar, does not have functionality for
contextual help or remnant icon logic (ie. remnant icon appearing after
viewing 2 videos).

carpeliam committed:
remove unused videos object

- videos are now within the objectPages object

carpeliam commented:
Hey @stevemanone! I was going to work on merging this PR, but it references some assets that aren't included in the source. e.g., `TapHoldOff.png`, and other assets mentioned in `App.js`. Can you add these to the PR?

bdougsand committed:
Merge remote-tracking branch 'stevemanone/GlobalNavBar'

bdougsand committed:
Merge help page into global nav branch

bdougsand committed:
Help page: fixed a silly mistake

stevemanone committed:
Misc bug fixes (YouTube videos are now working and can be watched, the
icons in the nav bar are no longer 'Button' components).  The YouTube
API link is not hard coded.

stevemanone committed:
Merge remote-tracking branch 'aashiyaan/GlobalNavBar' into GlobalNavBar

stevemanone committed:
Added code necessary to provide previous page information to help
component

bhinebaugh committed:
pass previous screen to help page via navigation params

bhinebaugh committed:
scroll help page to the section set in props from nav params

Keys of <Section> should match the names of the screens defined in TabNav,
currently:

screen      section
------      -------
Chooser --> Home (Objects)
RemnantChooser --> Remnants
About --> Resources

stevemanone committed:
Merge pull request #1 from bhinebaugh/master

scroll to section in help that corresponds to previous page
